RecoLocalCalo/EcalRecProducers/python/ecalRecHit_cfi.py

```diff
@@ -49,7 +49,7 @@
                             
     # for channel recovery
     algoRecover = cms.string("EcalRecHitWorkerRecover"),
-    recoverEBIsolatedChannels = cms.bool(False),
+                            recoverEBIsolatedChannels = cms.bool(True),##default is false
```

Comment:
Please keep the defaut "false" for 10_6 (and indent correctly)

Reply:
This is quite likely the reason of the differences observed in the test outpus

Reply:
I see. Done.